scripts/systemd/check_and_upgrade_serial_keel.sh

```diff
@@ -45,4 +45,4 @@ if $GIT_NO_NEW_REMOTE_CHANGES && $GIT_NO_BRANCH_SWITCH; then
 fi
 
 echo "Upgrading serial-keel.."
-cargo install --bin serial-keel --path $REPO_ROOT/core --features mocks-share-endpoints
+cargo install --bin serial-keel --path $REPO_ROOT/core
```

Comment:
Can I change this to $HOME/.cargo/bin/cargo install --path ... ? (for testing purposes)